backend/app/Http/Controllers/Auth/RegisteredUserController.php

```diff
@@ -0,0 +1,43 @@
+<?php
```

Comment:
if the controller's purpose is to handle incoming registration, then we can name it to RegisterController instead